data/unit_tests/make_mock_dataset.py

```diff
@@ -0,0 +1,90 @@
+import json
+from random import randint, uniform
+from typing import Any
+
+from faker import Faker
+from faker_airtravel import AirTravelProvider
+
+fake = Faker()
+fake.add_provider(AirTravelProvider)
+
+MAX_ALTITUDE = 13325
+MAX_RUNWAY_FT = 1000.0
+MAX_RUNWAY_M = 500.0
+MIN_RUNWAY_FT = 200.0
+MIN_RUNWAY_M = 100.0
+MAX_NUMBER_FLIGHTS = 500
+MAX_NUMBER_RUNWAYS = 16
+MAX_NUMBER_AIRPORTS = 100
+
+
+def fake_airports_generator(length: int) -> list[dict[str, Any]]:
+    airports_list = []
+    for _ in range(length):  # xrange in Python 2.7
+        og_iata = fake.airport_iata()
+        og_icao = fake.airport_icao()
```

Comment:
How do we ensure the `icao` and `iata` are consistent with one another?

Reply:
icao is just a placeholder we dont use it :/